crates/dotrain/src/composer/mod.rs

```diff
@@ -1389,4 +1405,35 @@ _: opcode-1(0xabcd 456);
 
         Ok(())
     }
+
+    /// [crate::types::patterns::E_PATTERN] and [crate::types::patterns::INT_PATTERN] but
+    /// without anchors/boundries (in this case they are ^ and $) which proptest doesnt support
+    const PATTERN: &str = r"[0-9]+(\.[0-9]+)?|[1-9][0-9]*(\.[0-9]+)?e[0-9]+";
```

Comment:
it's not really helpful if you use a regex to test itself

you need to use a different way to generate strings than the thing you're testing